### DISCUSSION TITLE
Backup, wireguard config should not be included

### DISCUSSION THREAD

**Discussion by iulko** [celzero/rethink-app#1652] [General]
Just to clarify for everybody, the good practice should be that wireguard config should not be included in backup or be only as an option to include it. Backup is not encrypted so should not include wireguard config. Just asking so that everybody knows.

**Comment by ignoramous** [comment]
It isn't: #1280 

Note that, (and afaik) the WireGuard official app for Android does not encrypt configurations with its own keys (relies on Android's built-in Filesystem encryption-at-rest). Rethink does (but I'll admit there's not much added security benefit to it).
